<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Company</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>22</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="horizontalSpacing">
     <number>4</number>
    </property>
    <item row="0" column="0" colspan="6">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(71, 71, 71);</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>0</y>
         <width>211</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;

color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Company Creation</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Company Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_16">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="leComapnyName"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Financial year beginning from</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_28">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QDateEdit" name="deFYDate"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Mailing Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_17">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="leMailingName"/>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_13">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Book beginning from</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_29">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QDateEdit" name="deBookYear"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_18">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPlainTextEdit" name="ptAddress"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>State</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_19">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="leState"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_20">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="leCountry"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Pincode</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_21">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="lePincode"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Mobile</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_22">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="leMobile"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Fax</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_23">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLineEdit" name="leFax"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_24">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QLineEdit" name="leEmail"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Website</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_25">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLineEdit" name="leWebsite"/>
    </item>
    <item row="11" column="0" colspan="6">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Base currency symbol</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="label_26">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QLineEdit" name="leCurrencySymbol"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_15">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Formal name</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="label_27">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QLineEdit" name="leFormalName"/>
    </item>
    <item row="14" column="4">
     <widget class="QPushButton" name="pbSubmit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(71, 71, 71);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;

color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
